pkg/memory-monitor/README.md

```diff
@@ -177,6 +177,25 @@ The tool than creates the output files in `/persist/memory-monitor/output`.
 The persistent storage is mounted to the container, so the output files are
 available in the host system.
 
+### Disabling the memory monitor
```

Comment:
Shouldn't this be reworded to be about enabling it since it is disabled by default?

Reply:
Will fix

Reply:
Done